src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/operations/blob.py

```diff
@@ -73,14 +73,15 @@ def action_blob_copy(blob_name):
             if dryrun:
                 logger.warning('  - copy blob %s', blob_name)
             else:
-                return _copy_blob_to_blob_container(client, source_client, destination_container,
+                return _copy_blob_to_blob_container(client, source_client, destination_container, destination_path,
                                                     source_container, source_sas, blob_name)
 
         return list(filter_none(action_blob_copy(blob) for blob in collect_blobs(source_client,
                                                                                  source_container,
                                                                                  pattern)))
 
     elif source_share:
+        import os.path
```

Comment:
Move this to the beginning of the file. No need to hide a basic import in the middle of the function. What I did previously was wrong.